index.html

```diff
@@ -388,9 +388,101 @@ <h2 id="protocol-registry">
     <h3>
       Inclusion criteria
     </h3>
+    <aside class="note">
+      The below criteria are a work in progress and are likely to change as
+      this document evolves.
+    </aside>
+    <p>
+      To be included in the registry, the [=digital credential/exchange
+      protocol=]:
+    </p>
+    <ol>
+      <li>MUST be standardized at a recognized standards organization and have
+      a stable URL.
+      </li>
+      <li>MUST define a [[WebIDL]] [=dictionary=] representation of the
+      [=digital credential/exchange protocol=] request structure (i.e., the
+      [=dictionary=] to which the {{DigitalCredentialsProvider}}'s
+      {{DigitalCredentialsProvider/request}} is [=converted to idl
+      values|converted=] to before it is passed onto underlying platform).
+      </li>
+      <li>MUST define a [[WebIDL]] [=dictionary=] representation of the
+      [=digital credential/exchange protocol=] response structure (i.e., the
+      [=dictionary=] to which the {{DigitalCredential}}'s
+      {{DigitalCredential/data}} is [=converted to idl values|=] to before it
+      is made available to the relying party).
+      </li>
+      <li>MUST define validation rules for members of the request and response
+      structures.
+      </li>
+      <li>MUST have undergone privacy review by the W3C's Privacy Interest
+      Group and Federated Identity Working Group.
+      </li>
+      <li>MUST have undergone security review by the Federated Identity Working
+      Group.
+      </li>
```

Comment:
Do we want to define the criteria by which this group would review those?

Reply:
Yes. I think so. I was hoping to let those folks chime in about how we might do it, as some are already members of this group.